.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,50 @@
+name: Create Release Pull Request
```

Comment:
Hi @PeterYinusa ,
Do you currently create the release branch `Version-vx.y.z` manually?
If yes, do you think this Github action could be used instead?

The reason why I'm asking is because by using this Github action, you would have to provide:
- the `x.y.z` version number for the current release, that will be used to create the `Version-vx.y.z` branch
- the next `x.y.z` version number for the next release that will be used to define the `release-x.y.z` label that will be applied on every future PR merged in main

Reply:
Hi @gauthierpetetin,

Yes, the release branch is created manually, but then there's some automation in place to create the PR

See: https://github.com/MetaMask/metamask-extension/blob/develop/.circleci/scripts/release-create-release-pr.sh

@danjm @Gudahtt may be able to provide more information on this.

Reply:
Thanks @PeterYinusa that's helpful.

I'll resolve this comment as after. discussion with @danjm on Friday we've decided to go for a different approach and define the next version number based on the following:
- Take the highest version number, based on the list of release branches (branches with `Version-vx.y.z` format)
- Take the version number indicated in package.json
- Use the highest version number amongst both values, and do a minor version bump to deduce the next version number